client/src/styles/Geocoder.css

```diff
@@ -0,0 +1,79 @@
+.leaflet-left {
+  position: fixed;
+  top: 70%;
+  right: 1%;
+}
+
+.leaflet-right {
+  margin: 1rem 0;
+}
+.leaflet-bar {
+  border-radius: 0.8rem;
+}
+
+.leaflet-control-geocoder-form {
+  display: block;
+  padding: 0.82rem;
+  border: 2px solid var(--secondary-color);
+  padding: 1.2rem 0.5rem 1rem 2rem;
+  border-radius: 0.6rem;
+}
+
+.leaflet-touch .leaflet-bar .leaflet-right {
+  display: flex;
+  align-items: center;
+  border: 0.17rem solid var(--fifth-color);
+  width: 102%;
+}
+
+::placeholder {
+  color: var(--secondary-color);
+  font-size: 0.7rem;
+  opacity: 0.9;
+}
+
+.leaflet-touch .leaflet-control-geocoder-icon {
+  position: absolute;
+  right: 0;
+  height: 3.5rem;
+  width: 3.2rem;
+  background-color: var(--secondary-color);
+  background-image: url("../assets/images/icons8-search.svg");
+}
```

Comment:
You can delete the "height: 3.5rem" and add at the end, just after "background-image" :
`  border-top-right-radius: 0.8rem;
  border-bottom-right-radius: 0.8rem;`

Reply:
done